io.github.nemuelw.proton-desktop.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.nemuelw.proton-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: proton-desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
all 3 is wrong, if it works on wayland use wayland+fallback-x11, if not only x11